frontend/src/store/FamLoginUserState.ts

```diff
@@ -106,6 +110,21 @@ const getApplicationsUserAdministers = () => {
     return applicationList;
 };
 
+const isAdminOfSelectedApplication = () => {
+    const userAdminAccess = getUserAccess().find(
```

Comment:
this looks good now, just this warning
<img width="649" alt="Screen Shot 2024-02-28 at 9 59 16 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/2906d556-19e5-4e50-a618-c3dadae52ae3">

Reply:
I forgot to put the "?", thanks!